src/lib/util.js

```diff
@@ -1,8 +1,8 @@
 export function isValidRepoUrl(url) {
-  url = url.trim();
+  url = url.toLowerCase().trim();
   url = url.substr(0, 1) === "/" ? url.substr(1) : url;
```

Comment:
@arth2002 I just want you to know you're being very patient with me! That said, I'd suggest we take a moment to change `url.substr()` to be `url.substring()`. It looks like `substr()` is deprecated (though widely supported). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr#browser_compatibility

Reply:
Done :+1: